src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -400,6 +387,43 @@ def persist_node_run(self, run_info: NodeRunInfo) -> None:
         filename = f"{str(line_number).zfill(self.LINE_NUMBER_WIDTH)}.jsonl"
         node_run_record.dump(node_folder / filename, run_name=self._run.name)
 
+    def _load_run_info(self, load_node: bool, parse_const_as_str: bool = False):
```

Comment:
Let's not do this, this is over-designed simplification, it would be really hard to maintain.

Reply:
For this particular case, please try defining a `load_file` function with class type as input parameter, and other logic should stay separated in different functions.